synapse_data_warehouse/synapse/dynamic_tables/V2.35.3__replace_node_latest.sql

```diff
@@ -0,0 +1,61 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+CREATE OR REPLACE DYNAMIC TABLE NODE_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+        WITH latest_unique_rows AS (
+            SELECT
+                CHANGE_TYPE,
+                CHANGE_TIMESTAMP,
+                CHANGE_USER_ID,
+                SNAPSHOT_TIMESTAMP,
+                ID,
+                BENEFACTOR_ID,
+                PROJECT_ID,
+                PARENT_ID,
+                NODE_TYPE,
+                CREATED_ON,
+                CREATED_BY,
+                MODIFIED_ON,
+                MODIFIED_BY,
+                VERSION_NUMBER,
+                FILE_HANDLE_ID,
+                NAME,
+                IS_PUBLIC,
+                IS_CONTROLLED,
+                IS_RESTRICTED,
+                SNAPSHOT_DATE,
+                EFFECTIVE_ARS,
+                ANNOTATIONS,
+                DERIVED_ANNOTATIONS,
+                VERSION_COMMENT,
+                VERSION_LABEL,
+                ALIAS,
+                ACTIVITY_ID,
+                COLUMN_MODEL_IDS,
+                SCOPE_IDS,
+                ITEMS,
+                REFERENCE,
+                IS_SEARCH_ENABLED,
+                DEFINING_SQL,
+                INTERNAL_ANNOTATIONS,
+                VERSION_HISTORY,
+                PROJECT_STORAGE_USAGE
```

Comment:
I made this `SELECT` explicit instead of using `SELECT *` since that would be helpful in knowing how exactly the schema changes for `node_latest` every time we re-introduce it when a new column gets added etc.